CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 None
 
+## [v1.1.2.2](https://github.com/pbrisbin/yesod-paginator/compare/v1.1.2.1...v1.1.2.2)
```

Comment:
Is this fine? I'm not sure how the release process works.

Reply:
Looks right to me. The only other (annoying) thing is you need to update the _Unreleased_ entry above so it compares from `v1.1.2.2` now.